backend/src/v2/driver/driver.go

```diff
@@ -940,3 +1021,203 @@ func generateOutputURI(root, artifactName string, taskName string) string {
 	// contains scheme like gs:// and path.Join cleans up scheme to gs:/
 	return fmt.Sprintf("%s/%s", strings.TrimRight(root, "/"), path.Join(taskName, artifactName))
 }
+
+var accessModeMap = map[string]corev1.PersistentVolumeAccessMode{
+	"ReadWriteOnce":    corev1.ReadWriteOnce,
+	"ReadOnlyMany":     corev1.ReadOnlyMany,
+	"ReadWriteMany":    corev1.ReadWriteMany,
+	"ReadWriteOncePod": corev1.ReadWriteOncePod,
+}
+
+func CreatePVC(inputs *pipelinespec.ExecutorInput_Inputs, namespace string) (pvcName string, err error) {
```

Comment:
NIT: add comment describing the function

Reply:
For this function, I think its name is pretty self-explanatory.  WDYT?